custom-metrics-stackdriver-adapter/pkg/adapter/translator/utils/filter_builder.go

```diff
@@ -107,6 +113,8 @@ func NewFilterBuilder(resourceType string) FilterBuilder {
 	switch resourceType {
 	case PodType:
 		schema = PodSchema
+	case PodContainerType:
```

Comment:
What if we named this something like `PodOrContainerType`?

Otherwise this gets confusing :(